katas/metal-slug/rubiana/characters.js

```diff
@@ -0,0 +1,46 @@
+export class Character {
+    constructor(_character, _hole, _habilities, _health, _vehicle) {
+        this.character = _character;
+        this.hole = _hole;
+        this.habilities = _habilities;
+        this.health = _health;
+        this.vehicle = _vehicle;
+    }
+    //shoot a weapon, use a knife and throw granades
+    characters = [{
```

Comment:
Not sure if this property will actually fit with the hierarchy you are defining some lines below. You are saying that `Character` will have `characters`, which at the same time is going to extended class inherit this and say something like "A soldier has characters, one soldier and one enemy". In such case, for reflecting the instances, I'd suggest to create instances of the class instead, and use tests for describing their attributes and behavior